src/pdf_beamtime/launch/pick_place.launch.py

```diff
@@ -0,0 +1,21 @@
+from launch import LaunchDescription
+from launch.substitutions import PathJoinSubstitution
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+
+def generate_launch_description():
+    action_cmd = Node(
+        package="pdf_beamtime",
+        executable="pick_place_action_server",
+        parameters=[
+            PathJoinSubstitution([FindPackageShare("pdf_beamtime"), "config", "pick_place_repeat_params.yaml"]),
```

Comment:
If the name of this default param file changes as suggested, it should be reflected here.